### ISSUE TITLE
why my requset say ,Not response many reqsuet ?

### ISSUE THREAD

**Issue by rebazomar121** [plainheart/bing-translate-api#10]
I'm blocked when I use bing-translator-API, so I try using a proxy for my server but it uses (got) to send API, and don't have an idea how to set proxy with this (got).

**Comment by plainheart** [comment]
Hi, I think it may be better to expose the `proxy` option of `got`. Will do it later.

**Comment by plainheart** [comment]
It's a bit strange that `agent` doesn't work. I saw you set `HttpProxyAgent` rather than `HttpsProxyAgent` as `https` agent, is it a typo?

**Comment by plainheart** [comment]
Released [v2.4.0](https://www.npmjs.com/package/bing-translate-api/v/2.4.0) on npm.

**Comment by rebazomar121** [comment]
thank for you 
it doesn't work for me so I try this code 
```
const { bootstrap } = require('global-agent');
process.env.GLOBAL_AGENT_HTTP_PROXY = proxyUrl;
process.env.NODE_TLS_REJECT_UNAUTHORIZED = false;
bootstrap();
```
after that I did write this inside got 
```
    https: {
      rejectUnauthorized: false
    },
```
now to work.......
